spec/models/knight_spec.rb

```diff
@@ -12,77 +8,77 @@ def find_piece(x, y)
   describe 'valid_move?' do
     before :all do
       @game = FactoryGirl.create(:game)
-      @bknight1 = find_piece(1, 7)
-      @bknight2 = find_piece(6, 7)
-      @wknight1 = find_piece(1, 0)
-      @wknight2 = find_piece(6, 0)
-      @bpawn1 = find_piece(0, 6)
-      @bpawn1.update(y_position: 5)
-      @bpawn4 = find_piece(3, 6)
-      @bpawn5 = find_piece(4, 6)
-      @bpawn8 = find_piece(7, 6)
-      @bpawn8.update(y_position: 5)
-      @wpawn1 = find_piece(0, 1)
-      @wpawn1.update(y_position: 2)
-      @wpawn4 = find_piece(3, 1)
-      @wpawn5 = find_piece(4, 1)
-      @wpawn8 = find_piece(7, 1)
-      @wpawn8.update(y_position: 2)
+      @bknight1 = find_piece(1, 0)
+      @bknight2 = find_piece(6, 0)
+      @wknight1 = find_piece(1, 7)
+      @wknight2 = find_piece(6, 7)
+      @bpawn1 = find_piece(0, 1)
+      @bpawn1.update(y_position: 2)
+      @bpawn4 = find_piece(3, 1)
+      @bpawn5 = find_piece(4, 1)
+      @bpawn8 = find_piece(7, 1)
+      @bpawn8.update(y_position: 2)
+      @wpawn1 = find_piece(0, 6)
+      @wpawn1.update(y_position: 5)
+      @wpawn4 = find_piece(3, 6)
+      @wpawn5 = find_piece(4, 6)
+      @wpawn8 = find_piece(7, 6)
+      @wpawn8.update(y_position: 5)
```

Comment:
all the changes here in the tests are to swap black pieces to the top of the board and white to the bottom